tuyaha/tuyaapi.py

```diff
@@ -33,13 +40,47 @@ class TuyaSession:
 
 class TuyaApi:
 
+    def __init__(self, log_level_warn=True):
+        self._log_level_warn = log_level_warn
```

Comment:
This is not true. Currently discovery (as also query) never fail if API polling interval is respected, so warning is more correct because means that you are pollind to fast. But because I'm not confidant with Tuya API, I prefer to wrap the method to be able to disable warning if somenthing change.